lib/config/presets/internal/custom-managers.ts

```diff
@@ -122,7 +117,7 @@ export const presets: Record<string, Preset> = {
     customManagers: [
       {
         customType: 'regex',
-        fileMatch: ['.+\\.tfvars$'],
+        filePatterns: ['/.+\\.tfvars$/'],
```

Comment:
```suggestion
        filePatterns: ['**/*.tfvars'],
```
simplify others too where possible